Doc/library/runpy.rst

```diff
@@ -137,22 +138,22 @@ The :mod:`runpy` module provides two functions:
    supplied path, and ``__spec__``, ``__cached__``, ``__loader__`` and
    ``__package__`` will all be set to :const:`None`.
 
-   If the supplied path is a reference to a valid sys.path entry, then
+   If the supplied path is a reference to a valid :data:`sys.path` entry, then
    ``__spec__`` will be set appropriately for the imported ``__main__``
    module (that is, ``__spec__.name`` will always be ``__main__``).
    ``__file__``, ``__cached__``, ``__loader__`` and ``__package__`` will be
    :ref:`set as normal <import-mod-attrs>` based on the module spec.
 
    A number of alterations are also made to the :mod:`sys` module. Firstly,
-   ``sys.path`` may be altered as described above. ``sys.argv[0]`` is updated
-   with the value of ``path_name`` and ``sys.modules[__name__]`` is updated
-   with a temporary module object for the module being executed. All
+   :data:`sys.path` may be altered as described above. ``sys.argv[0]``
+   is updated with the value of ``path_name`` and ``sys.modules[__name__]`` is
+   updated with a temporary module object for the module being executed. All
```

Comment:
We normally prefer to keep the diff as small as possible, to keep `git blame` easy to read. Here you're altering 3 lines, but we could do the same thing just by altering one line. (It doesn't matter if the line length goes _slightly_ over 80 characters :)

```suggestion
   :data:`sys.path` may be altered as described above. ``sys.argv[0]`` is updated
   with the value of ``path_name`` and ``sys.modules[__name__]`` is updated
   with a temporary module object for the module being executed. All
```

Reply:
Ah, yes, I persistently tried not to exceed 80 characters 😄 But what you say makes sense. I'll push an update.